README.md

```diff
@@ -22,69 +22,69 @@ We encourage you to participate in this open source project. We love Pull Reques
 
 High-level components for building browser(-like) apps.
 
-* 🔵 **Domains** Localized and customizable domain lists for auto-completion in browsers.
+* 🔵 **Domains** - Localized and customizable domain lists for auto-completion in browsers. [![Bintray](https://img.shields.io/bintray/v/pocmo/Mozilla-Mobile/domains.svg)](https://bintray.com/pocmo/Mozilla-Mobile/domains)
 
-* 🔴 **Engine-Gecko** - *Engine* implementation based on [GeckoView](https://wiki.mozilla.org/Mobile/GeckoView).
+* 🔴 **Engine-Gecko** - *Engine* implementation based on [GeckoView](https://wiki.mozilla.org/Mobile/GeckoView). [![Bintray](https://img.shields.io/bintray/v/pocmo/Mozilla-Mobile/engine-gecko.svg)](https://bintray.com/pocmo/Mozilla-Mobile/engine-gecko)
```

Comment:
Linking to the bintray repository may be confusing, given that we want consumers to pull from JCenter.

Reply:
I thought eventually we want to move to "our" Bintray repository, then we can simply change the link. JCenter doesn't have a landing page for each package. I agree though that this doesn't seem to load reliably. So, we're better off just documenting the Gradle coords in individual READMEs.